plugins/modules/proxmox.py

```diff
@@ -589,137 +844,184 @@ def create_instance(self, vmid, node, disk, storage, cpus, memory, swap, timeout
         # Fail on unsupported features
         for option, version in minimum_version.items():
             if pve_version < LooseVersion(version) and option in kwargs:
-                self.module.fail_json(changed=False, msg="Feature {option} is only supported in PVE {version}+, and you're using PVE {pve_version}".
-                                      format(option=option, version=version, pve_version=pve_version))
-
-        if VZ_TYPE == 'lxc':
-            kwargs['cpulimit'] = cpus
-            kwargs['rootfs'] = disk
-            if 'netif' in kwargs:
-                kwargs.update(kwargs['netif'])
-                del kwargs['netif']
-            if 'mounts' in kwargs:
-                kwargs.update(kwargs['mounts'])
-                del kwargs['mounts']
-            if 'pubkey' in kwargs:
-                if self.version() >= LooseVersion('4.2'):
-                    kwargs['ssh-public-keys'] = kwargs['pubkey']
-                del kwargs['pubkey']
+                self.module.fail_json(
+                    changed=False,
+                    msg="Feature {option} is only supported in PVE {version}+, and you're using PVE {pve_version}".format(
+                        option=option, version=version, pve_version=pve_version
+                    ),
+                )
+
+        if VZ_TYPE == "lxc":
+            kwargs["cpulimit"] = cpus
+            kwargs["rootfs"] = disk
+            if "netif" in kwargs:
+                kwargs.update(kwargs["netif"])
+                del kwargs["netif"]
+            if "mounts" in kwargs:
+                kwargs.update(kwargs["mounts"])
+                del kwargs["mounts"]
+            if "pubkey" in kwargs:
+                if self.version() >= LooseVersion("4.2"):
+                    kwargs["ssh-public-keys"] = kwargs["pubkey"]
+                del kwargs["pubkey"]
         else:
-            kwargs['cpus'] = cpus
-            kwargs['disk'] = disk
+            kwargs["cpus"] = cpus
+            kwargs["disk"] = disk
```

Comment:
There is **a lot** of unrelated reformatting in this PR. Can you please remove that? It makes reviewing it unnecessary hard.

Reply:
This *should* be fixed now. Sorry about that. VS Code auto-formats files for me, but luckily I had it as a separate commit.